lambda-handler.js

```diff
@@ -1,23 +1,31 @@
-var tachyon = require( './index' )
+var tachyon = require('./index');
 
-exports.handler = function( event, context ) {
-	var region = process.env.S3_REGION
-	var bucket = process.env.S3_BUCKET
-	var key = event.key
-	var args = event.args
-	return tachyon.s3( { region: region, bucket: bucket }, key, args, function( err, data, info ) {
-		if ( err ) {
-			context.fail( err )
-		} else {
-			context.succeed( {
-				data: new Buffer( data ).toString( 'base64' ),
-				format: info.format,
-				size: info.size
-			} )
-		}
+exports.handler = function(event, context, callback) {
+  var region = process.env.S3_REGION;
+  var bucket = process.env.S3_BUCKET;
+  var key = event.path.substring(1);
+  var args = event.queryStringParameters;
+  return tachyon.s3(
```

Comment:
Prettier all the things! I'm thinking a shortcut to run Prettier + convert to tabs :D